### ISSUE TITLE
Multi-line literal strings + CRLF line endings = False positive 'Invalid character in string'

### ISSUE THREAD

**Issue by rhkleijn** [tamasfe/taplo#11]
On my Windows machine the Even Better TOML extension for VS Code reports a bunch of 'Invalid character in string' problems with the multi-line literal string (three single quotes) in this example pyproject.toml file taken from [https://black.readthedocs.io/en/stable/pyproject_toml.html](https://black.readthedocs.io/en/stable/pyproject_toml.html).

```
[tool.black]
line-length = 88
target-version = ['py37']
include = '\.pyi?$'
exclude = '''

(
  /(
      \.eggs         # exclude a few common directories in the
    | \.git          # root of the project
    | \.hg
    | \.mypy_cache
    | \.tox
    | \.venv
    | _build
    | buck-out
    | build
    | dist
  )/
  | foo.py           # also separately exclude a file named foo.py in
                     # the root of the project
)
'''
```

I suspect it has to do with line endings. When I convert from CRLF to LF the extension does not  report any problems in this file.

**Comment by tamasfe** [comment]
Ah I knew I'd eventually run into some CRLF related bug. Thanks! I'll look into it in the following days.

**Comment by tamasfe** [comment]
Apparently all CR characters are denied in multi-line strings, while they shouldn't be. It'll be fixed in the next version among a bunch of other fixes. I'll try to release it this weekend.

**Comment by tamasfe** [comment]
This should be fixed in 0.5.1 now.
